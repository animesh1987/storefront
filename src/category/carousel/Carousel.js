import React from 'react';

// Set up style for background image for carousel image.
const backgroundImage = {
  height: '35vh',
  background: `url(${process.env.PUBLIC_URL}/media/plates-header.jpg)`,
  backgroundSize: 'cover'
};

export const Carousel = () => {
  return (
    <div className="Category__carousel"
      style={ backgroundImage }>
      <div className="Category__tagline-container">
        <div className="Category__tagline">
          <span className="App-title">Plates</span>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
            Nam at ante elementum, porta neque ac, accumsan erat.
            Etiam sit amet ligula vitae ipsum volutpat vestibulum.
            Ut quis velit lorem. Nunc placerat nec.
          </p>
        </div>
      </div>
    </div>
  );
}
