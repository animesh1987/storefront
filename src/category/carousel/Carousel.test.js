import React from 'react';
import { create } from 'react-test-renderer';
import { Carousel } from './Carousel';
 
it('renders product page correctly', () => {
  const component = create(<Carousel />)
    .toJSON();
  expect(component).toMatchSnapshot();
});
