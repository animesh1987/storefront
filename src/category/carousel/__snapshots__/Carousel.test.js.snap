// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders product page correctly 1`] = `
<div
  className="Category__carousel"
  style={
    Object {
      "background": "url(/media/plates-header.jpg)",
      "backgroundSize": "cover",
      "height": "35vh",
    }
  }
>
  <div
    className="Category__tagline-container"
  >
    <div
      className="Category__tagline"
    >
      <span
        className="App-title"
      >
        Plates
      </span>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam at ante elementum, porta neque ac, accumsan erat. Etiam sit amet ligula vitae ipsum volutpat vestibulum. Ut quis velit lorem. Nunc placerat nec.
      </p>
    </div>
  </div>
</div>
`;
